src/rgw/rgw_sal_motr.cc

```diff
@@ -3442,14 +3442,36 @@ void *newMotrStore(CephContext *cct)
     const auto& ha_ep    = g_conf().get_val<std::string>("motr_ha_endpoint");
     const auto& proc_fid = g_conf().get_val<std::string>("motr_my_fid");
     const auto& profile  = g_conf().get_val<std::string>("motr_profile_fid");
+    const auto& admin_proc_ep  = g_conf().get_val<std::string>("admin_motr_endpoint");
+    const auto& admin_proc_fid = g_conf().get_val<std::string>("admin_motr_fid");
+    const int init_flags = cct->get_init_flags();
     ldout(cct, 0) << "INFO: motr my endpoint: " << proc_ep << dendl;
     ldout(cct, 0) << "INFO: motr ha endpoint: " << ha_ep << dendl;
     ldout(cct, 0) << "INFO: motr my fid:      " << proc_fid << dendl;
     ldout(cct, 0) << "INFO: motr profile fid: " << profile << dendl;
     store->conf.mc_local_addr  = proc_ep.c_str();
     store->conf.mc_process_fid = proc_fid.c_str();
+
+    ldout(cct, 0) << "INFO: init flags:        " << init_flags << dendl;
+    ldout(cct, 0) << "INFO: admin motr endpoint:  " << admin_proc_ep << dendl;
```

Comment:
remove these 3456, 3457 logs as line 3470-3473 will any way print final FIDS used for motr init.